src/assets/url.js

```diff
@@ -1,4 +1,5 @@
 const SAPMLE_USER = "https://bootcamp-api.codeit.kr/api/sample/user";
 const SAMPLE_FOLDER = "https://bootcamp-api.codeit.kr/api/sample/folder";
+const CODEIT_API = "https://bootcamp-api.codeit.kr/api";
```

Comment:
```suggestion
export const CODEIT_API = "https://bootcamp-api.codeit.kr/api";
export const SAPMLE_USER = `${CODEIT_API}/sample/user`;
export const SAMPLE_FOLDER = `${CODEIT_API}/sample/folder`;
```
[중요도: 중간] 상수화 좋습니다! 다만 상수화를 재활용하기 위해 이런 방식도 추천드립니다.
[중요도: 중간] export를 최하단에 몰아서 선언하는 것보다 바로 옆에 붙이는 것도 좋은 방법입니다. 최하단 선언의 단점은 50-60개가 넘어가게 되면(혹은 그 전에도 그럴 수 있음) 맨 밑에 찾아서 적기 귀찮고 여러명이 이 파일을 건드리게 되면 conflict이 날 가능성도 있습니다~